Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/competition/BlueRight.java

```diff
@@ -0,0 +1,94 @@
+package org.firstinspires.ftc.team6220_PowerPlay.competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+
+@Autonomous(name = "BlueRight")
+public class BlueRight extends BaseAutonomous {
```

Comment:
BlueRight seems to have a lot of similarities to BlueLeft. It would be great to create functions that call the smaller functions so you can reuse all that code.

Reply:
We have separate auton method for each quadrant of the field to help drivers

NOTE:

We might generalize the code inside these classes into a method soon, and we will implement this for league 5 (worlds)